pkg/composableschemadsl/parser/tests/localimport_keyword_in_identifiers.zed.expected

```diff
@@ -0,0 +1,121 @@
+NodeTypeFile
+  end-rune = 154
+  input-source = local imports with keyword in identifiers test
+  start-rune = 0
+  child-node =>
+    NodeTypeImport
+      end-rune = 30
+      input-source = local imports with keyword in identifiers test
+      start-rune = 0
+      child-node =>
+        NodeTypeError
+          end-rune = 30
+          error-message = Expected one of: [TokenTypeComma], found: TokenTypeKeyword
+          error-source = caveat
+          input-source = local imports with keyword in identifiers test
+          start-rune = 32
```

Comment:
Why doesn't this line up with the error below?

Reply:
This typically is a result of missing a finishNode call somewhere, but I don't see it. You'll need to trace to see why that is